configs/mmdet3d/mono-detection/mono-detection_tensorrt_dynamic-320x320-1344x1344.py

```diff
@@ -0,0 +1,11 @@
+_base_ = ['./mono-detection_dynamic.py', '../../_base_/backends/tensorrt.py']
+backend_config = dict(
+    common_config=dict(max_workspace_size=1 << 30),
+    model_inputs=[
+        dict(
+            input_shapes=dict(
+                imgs=dict(
```

Comment:
better change `imgs` ad `input` as it is mostly used.

Reply:
ok